src/containers/Caregroup/Caregroup.js

```diff
@@ -1,13 +1,18 @@
 import React, { Component } from 'react';
-import CareGroupCard from '../../components/Card/caregroup-card';
+import CaregroupCard from '../../components/Card/CaregroupCard';
+
+import './Caregroup.css';
 
 export default class CareGroup extends Component {
-    render() {
-        return (
-            <div>
-                <h1>Care Group</h1>
-                <CareGroupCard/>
-            </div>
-        );
-    }
+  render() {
+    return (
+      <React.Fragment>
+        <h1>Care Group</h1>
+        <div className='caregroup'>
+          <CaregroupCard />
```

Comment:
Hmm this is weird why you dont put the map of the states in here instead? The right approach is to make CareGroupCard a function component instead that receives props and just render it without business logic. You should put the business logic in this component instead